src/Core/Manifest/Module.php

```diff
@@ -19,6 +20,23 @@ class Module implements Serializable
      */
     const TRIM_CHARS = ' /\\';
 
+    /**
+     * Return value of getCILibrary() when module uses PHPUNit 9
+     */
+    const CI_PHPUNIT_NINE = 'PHPUnit9';
+
+    /**
+     * Return value of getCILibrary() when module uses PHPUNit 5
+     */
+    const CI_PHPUNIT_FIVE = 'PHPUnit5';
+
+    /**
+     * Return value of getCILibrary() when module does not use any CI
+     */
+    const CI_PHPUNIT_UNKNOWN = 'NoPHPUnit';
```

Comment:
Maybe this should not reference PHPUnit at all.

```suggestion
    const CI_UNKNOWN = 'NoPHPUnit';
```

Reply:
Should just do the usual `const CI_UNKNOWN = 'CI_UNKNOWN';`

Reply:
Will do.

Reply:
Done